### DISCUSSION TITLE
hello everyone❤

### DISCUSSION THREAD

**Discussion by duongducbinh** [community/community#122525] [GitHub Education]
### Hi everyone,

I am a new member and hope to receive help from everyone😍

**Comment by davevad93** [comment]
Hi @duongducbinh and welcome to GitHub.

We're a very large community from all over the world, if you need help with anything open a discussion and you will surely find someone who can help you with your questions. 

Have a nice day!

**Comment by DeiviHerreraDiaz09** [comment]
# Hello my friends 🤟 

> [!NOTE]
>  I'm new, I'm just starting out in development, do you have any materials for a Full Stack Developer roadmap?

**Reply by MehmetBozkir** [reply]
Of course, you can go to the link below :)

Link : https://roadmap.sh/full-stack

**Comment by MehmetBozkir** [comment]
Hello, I can help you with FE, I am especially interested in React.

**Reply by ChineduJeremiahan** [reply]
hello
